requirements.txt

```diff
@@ -0,0 +1,3 @@
+dash
```

Comment:
Can you plz add the versions here?

Reply:
Yes sure ; I used the latest version for testing purposes on my machine and it worked straight out of the box. Some deprecation warnings though with visdcc, I'll stick with the latest stable releases

Reply:
Sure

Reply:
Should be ok by now, I let you review the latest patch and off we go ! :)